java/src/jmri/jmrix/sprog/update/Sprogv4UpdateAction.java

```diff
@@ -11,6 +11,7 @@
  *
  * @author	Andrew crosland Copyright (C) 2004
  */
+@Deprecated
```

Comment:
Please include a Javadoc comment like ```@deprecated since 4.11.1; supports uncommon Sprog versions that are confused with Sprog II versions```. This will let us know when it is okay to remove this class (if it should simply not be used but should also not be removed, the comment should be something like ```@deprecated since 4.11.1; not for removal; supports ...```

Note that Java versions after 9 are likely to begin requiring the ```@deprecated``` Javadoc comment when using the ```@Deprecated``` annotation.